src/menus/menu-item.jsx

```diff
@@ -33,6 +33,7 @@ const MenuItem = React.createClass({
     leftIcon: React.PropTypes.element,
     rightIcon: React.PropTypes.element,
     onTouchTap: React.PropTypes.func,
+    onClick: React.PropTypes.func,
```

Comment:
We shouldn't have any onClick property
This mistake is from
```js
if (this.props.onClick) {
      this.props.onClick(e);
    }
```
Could you replace the source code with `onTouchTap`?